src/components/FilesSideBar/FileMenuOption/DirectoryOption.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+} from "@mui/material";
+import React, { useState } from "react";
+import { FileNode } from "../FileNode";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import FileOption from "./FileOption";
+import FolderIcon from "@mui/icons-material/Folder";
+
+interface DirectoryOptionProps {
+  node: FileNode;
+  fileSelected: string;
+  setFileSelected: (data: string, path: string) => void;
+  root?: boolean;
+}
+
+const DirectoryOption: React.FC<DirectoryOptionProps> = ({
+  node,
+  fileSelected,
+  setFileSelected,
+  root = true,
+}) => {
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <Box>
+      <Accordion
+        disableGutters
+        sx={{
+          backgroundColor: "#2A2F40",
```

Comment:
For the bgcolor, use the colors in styleConstant, like:
<Typography color={COLORS.GREEN} >HOLAAAA</Typography>